dashboard/config/locales/scripts.en.yml

```diff
@@ -192,6 +197,7 @@ en:
           lessons:
             Maze:
               name: Maze
+          student_description: Try the basics of computer science with characters from Angry Birds, Plants vs. Zombies, and Scrat from Ice Age!
```

Comment:
what are the next steps here -- will `description` get updated with content from CB?

Reply:
I was going to leave that up to curriculum writers to decide what they want to do once these fields are live. For now this does no harm in that it keeps the same description showing that was there before. As editors work on 2021 they can figure out if they just want the unit descriptions from CB moved over or not. Then if we figure out a general plan when we import the other past courses we can follow that plan